superset/jinja_context.py

```diff
@@ -297,7 +297,7 @@ def get_filters(self, column: str, remove_filter: bool = False) -> List[Filter]:
 
         for flt in form_data.get("adhoc_filters", []):
             val: Union[Any, List[Any]] = flt.get("comparator")
-            op: str = flt["operator"].upper() if "operator" in flt else None
+            op: str = flt["operator"].upper() if "operator" in flt and flt["operator"] is not None else None
```

Comment:
We could probably simplify this to something like 
```suggestion
            op: str = flt["operator"].upper() if flt.get("operator") else None
```